templates/field/field--media--webtools-op-publication-list.html.twig

```diff
@@ -4,8 +4,4 @@
  * Theme override for the webtools op publication list fields on media items.
  */
 #}
-{% for item in items %}
-  {% include '@ecl-twig/media-container' with {
-    'embedded_media': item.content,
-  } %}
-{% endfor %}
+{%- include 'field--bare.html.twig' -%}
```

Comment:
if we delete the template altogether then we do get the field title and some attribute-less divs so I guess this is better.